docs/source/app_developers_guide/rust_sdk_import.rst

```diff
@@ -0,0 +1,48 @@
+************************
+Importing the Rust SDK
+************************
+
+.. note::
+   The Sawtooth Rust SDK assumes that you have the latest version of Rust and
+   its package manager Cargo, which can be installed with `rustup
+   <https://rustup.rs/>`_
+
+Once you've installed the Sawtooth Core, there are a few additional steps
+you'll need to take to get started developing for Sawtooth in Rust.
+
+1. Add Sawtooth to your ``Cargo.toml`` file. The Rust SDK is located within
+   the Sawtooth Core: ``/project/sawtooth-core/sdk/rust``.
+
+.. note::
+    You may need to change the path to the SDK to reflect where your working
+    directory is. For example, when developing the ``xo_rust`` transaction
+    processor (located in ``projet/sawtooth-core/sdk/examples/xo_rust``), one
+    would use the path ``../../rust``)
+
+.. code-block:: ini
+    :caption: Sample ``Cargo.toml`` for a Sawtooth Rust project
+
+    [package]
+    name = "package_name"
+    version = "0.1.0"
+    authors = ["..."]
+
+    [dependencies]
+    sawtooth_sdk = { path = "/project/sawtooth-core/sdk/rust"}
```

Comment:
This should be changed to use `sawtooth_sdk = { git = "https://github.com/hyperledger/sawtooth-core.git" }` instead of path